packages/js-dpp/lib/identity/validation/validatePublicKeysFactory.js

```diff
@@ -38,17 +45,34 @@ function validatePublicKeysFactory(validator, bls) {
    * @typedef validatePublicKeys
    *
    * @param {RawIdentityPublicKey[]} rawPublicKeys
+   * @param {{mustBeEnabled: boolean}} [options]
    *
    * @return {ValidationResult}
    */
-  function validatePublicKeys(rawPublicKeys) {
+  function validatePublicKeys(rawPublicKeys, options = {}) {
     const result = new ValidationResult();
 
+    if (rawPublicKeys.length > identitySchema.properties.publicKeys.maxItems) {
```

Comment:
We don't need to check it here since it will be checked by the parent JSON schema (identity, identity create transition)